src/Runner.Common/HostTraceListener.cs

```diff
@@ -89,10 +89,15 @@ public override void WriteLine(string message)
             }
 
             Flush();
+
+            if (StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.PrintLogToStdout)))
+            {
+                Console.WriteLine(message);
```

Comment:
I am wondering whether we should add a `ConsoleTraceListener` base on the setting instead of writing to the console within the `TextWriterTraceListener`.

Reply:
Thanks @TingluoHuang , changes have been added

Reply:
ConsoleTraceListener didn't format it, we implemented it in an `StdoutTraceListener` class instead